### ISSUE TITLE
Command Center should shrink when not enough space

### ISSUE THREAD

**Issue by Marc2p** [microsoft/vscode#236302]
When Vscode is in English the menu bar contains the items "file", "edit", "selection", "view", "go", "run", "terminal" and "help".
![Image](https://github.com/user-attachments/assets/ca2be6d6-b0a0-4bec-83f8-a3d495a9152e)
When I change the display language to Spanish it shows like this:
![Image](https://github.com/user-attachments/assets/46893226-fb7a-4bcc-8ccf-86cc0e0b7c45)

Best regards

**Comment by TylerLeonhardt** [comment]
Yeah we want the command center to be narrower in this case. cc @jrieken

**Comment by benibenj** [comment]
This issue will always exist on smaller screens. We could allow the command center to shrink a little bit in this case, but probably not by much.

**Comment by TylerLeonhardt** [comment]
Yeah I think we should shrink the Command Center. It's already quite big... and it wasn't a problem that it was so big before. Now it is so I think we can make it smaller so it's not as disruptive.

**Comment by ykgw-liuxin** [comment]
> When Vscode is in English the menu bar contains the items "file", "edit", "selection", "view", "go", "run", "terminal" and "help". ![Image](https://github.com/user-attachments/assets/ca2be6d6-b0a0-4bec-83f8-a3d495a9152e) When I change the display language to Spanish it shows like this: ![Image](https://github.com/user-attachments/assets/46893226-fb7a-4bcc-8ccf-86cc0e0b7c45)
> 
> Best regards

Agree.
Even in English, the command center is too wide in the default size window, causing the menu to collapse.
I want the command center to shrink or autoscale to show all menus in a default-sized window.

![Image](https://github.com/user-attachments/assets/c71c695a-c92f-4bdf-8ad5-2a9b4a786430)

**Comment by MichaelMagdyG** [comment]
When I launch VS Code, the full menu bar (`File`, `Edit`, `Selection`, `View`, `Go`, `Run`, `Terminal`, `Help`) is displayed correctly for a few seconds. Once the UI finishes loading, the menu bar is recalculated and the `Terminal` and `Help` entries disappear from the visible area.

This behavior occurs consistently on every startup. It appears to be a layout issue where the title bar or Command Center expands after initialization, reducing the available space for menu items. Instead of adapting or shrinking appropriately, the rightmost menu entries are pushed out of view.

Key observations:

* The issue occurs immediately after startup without any manual window resizing.
* It happens regardless of the initial window size.
* The menu items are visible briefly during startup, then disappear once the UI finishes loading.
* This suggests the menu itself is being rendered correctly initially and is later affected by a layout recalculation.

### Video Evidence

https://github.com/user-attachments/assets/fc1b8c30-1097-4016-ba09-3d3b4d042674

### Comparison

On the same machine, VS Code for the Web (`https://vscode.dev`) displays the complete menu bar correctly and does not exhibit this behavior.

<img width="1366" height="720" alt="VS Code menu bar issue" src="https://github.com/user-attachments/assets/96be523a-1255-417e-9569-5dafd47dc3e1" />

**Comment by psmithwork** [comment]
I experimented with the title-bar allocation in `titlebarpart.css`.

Rather than define the target by a specific pixel width, I used a normally sized VS Code window where the Status Bar still has enough room to display its standard information, including `Ln/Col`, encoding, line endings, and language mode. Once the window gets narrower than that, VS Code itself begins dropping Status Bar items, so that seemed like a reasonable boundary for this test.

At that size, my useful criteria were:

* Keep the complete `File ... Help` menu visible.
* Keep the workspace name readable in the Command Center.

Reducing the Command Center's `max-width` from 600px to 240px made no observable difference. The surrounding title-bar allocation was the limiting factor.

The current allocation is roughly:

```css
20% / 60% / 20%
left / center / right
```

After testing several ratios, I got much better use of the available width with:

```css
45% / 25% / 30%
left / center / right
```

With that allocation, the complete menu remains visible at a normal usable window size while the workspace name still has ample room, both with and without the Agents title-bar control enabled.

The Command Center already handles text overflow, so reserving 60% of the title bar for the center seems unnecessarily conservative when the left side contains menu items that otherwise get pushed into overflow.

Happy to submit a small PR if this seems like a useful direction; I wanted to post the experiment first since the preferred final ratio may differ.
